src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -423,6 +423,30 @@ cdef class Font:
         else:
             return None
 
+    def get_nominal_glyph(self, unicode: int):
+        cdef hb_codepoint_t gid
+        success = hb_font_get_nominal_glyph(self._hb_font, unicode, &gid)
+        return gid if success else None
+
+    def get_var_coords_normalized(self):
+        cdef unsigned int length
+        cdef const int *coords
+        coords = hb_font_get_var_coords_normalized(self._hb_font, &length)
+        return [coords[i] / 0x4000 for i in range(length)]
```

Comment:
Probably it is just my ignorance, but it is not clear from the code what this division is about (and the multiplication below), so a comment might help.

Reply:
I'll add a comment. It converts 2.14 to float.

Reply:
Done in master: 2f533d9d1131a2e630dfbe71956da6816db05785

Reply:
Thanks!